### PR TITLE
Added description for parameter keys in JSDoc in FxAccountClient

### PR DESCRIPTION
**Pull request by Ashwin02**

### CODE REVIEW

File: client/FxAccountClient.js [hunk 1]
```diff
@@ -871,6 +871,7 @@ define([
    * @param {String} email
    * @param {String} newPassword
    * @param {Object} oldCreds This object should consists of `oldUnwrapBKey`, `keyFetchToken` and `passwordChangeToken`.
+   * @param {Boolean} [options.keys] If `true`, calls the API with `?keys=true` to get the keyFetchToken
```
Comment (pdehaan) [inline client/FxAccountClient.js]: I think you may have a trailing space at the end of this line:

```
client/FxAccountClient.js
  874:105  error  Trailing spaces not allowed  no-trailing-spaces
```

Comment (vladikoff) [inline client/FxAccountClient.js]: Thanks for the pull request!

This line needs to be changed from:

```
* @param {Boolean} [options.keys] If `true`, calls the API with `?keys=true` to get the keyFetchToken 
```
to 
```
* @param {Object} keys This object should contain the unbundled keys
```

File: client/FxAccountClient.js [hunk 2]
```diff
@@ -871,6 +871,7 @@ define([
...
```
Comment (vladikoff) [inline client/FxAccountClient.js]: Resubmitting the comment for visibility: 

Thanks for the pull request!

This line needs to be changed from:
```
* @param {Boolean} [options.keys] If `true`, calls the API with `?keys=true` to get the keyFetchToken 
```
to

```
* @param {Object} keys This object should contain the unbundled keys
```

File: client/FxAccountClient.js [hunk 3]
```diff
@@ -1099,4 +1100,3 @@ define([
 
   return FxAccountClient;
 });
-
```
Comment (vladikoff) [inline client/FxAccountClient.js]: This space should not be removed.

**Comment by vladikoff** [comment]
Once you make the changes please git squash the commits into one commit with the following commit message:

```
fix(docs): Fix jsdoc for _passwordChangeFinish

Fixes #190
```

Thanks!

**Comment by Ashwin02** [comment]
Hello,
Sorry for all that fuss. I hope the document is as expected now. I think I may have messed up while git squashing the commits. If yes please let me know I will try to fix it.
Thank you for all the help.

**Comment by vladikoff** [comment]
@Ashwin02 that's okay, see the squashing guide here: http://push.cwcon.org/learn/squashing-commits.html

You currently have 3 commits and it needs to be just 1. :+1:

**Comment by vladikoff** [comment]
Thank you @Ashwin02 !

**Pull request merged**